tasks/messageexecutions/message.go

```diff
@@ -39,6 +38,12 @@ func (p *Task) ProcessMessageExecutions(ctx context.Context, store adt.Store, ts
 		StateRoot: pts.ParentState().String(),
 	}
 
+	mex, err := p.node.GetMessageExecutionsForTipSet(ctx, ts, pts)
+	if err != nil {
+		report.ErrorsDetected = xerrors.Errorf("getting messages executions for tipset: %w", err)
+		return nil, report, nil
+	}
+
```

Comment:
Returning the error here would be fatal meaning the corresponding walk, watch, or fill job would abort completely -- that would be inconsistent with current behavior. So, we list this as a detected error in the report which will mark the task as `ERROR` in the corresponding processing report, and return immediately with no data extracted.